### PR TITLE
Feature/manual stream mode minimal

### PR DESCRIPTION
**Pull request by SecondeJK**
This PR adds the ability to take a new endpoint value for both Archive and Broadcast to set the `streamMode`

Link to JIRA ticket:
https://jira.vonage.com/browse/DEVX-5713

## Description
Both Archive and Broadcast can now take a new value in the `POST` requests to create a new session named `streamMode`. This defaults to `auto`, but can be set to `manual`, allowing the developer to hit further endpoints to manually add or remove a stream to the session to customise it.

## Motivation and Context
This is a new feature that is due in the API, and therefore needs SDK support.

## How Has This Been Tested?
Two new tests have been included that follow the existing pattern of tests. On review, please note that these tests -assume- that the API response is request and DO NOT test the result of making a call using the internal methods of the SDK. These types of test do not exist in the codebase, but that will be due a large overhaul of the test suite when hopefully moving to a VCR record-and-playback style of API test integration.

## Types of changes

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/OpenTok/Broadcast.php [hunk 1]
```diff
@@ -34,6 +34,10 @@
 *
 * @property boolean $isStopped
 * Whether the broadcast is stopped (true) or in progress (false).
+*
+* @property string $streamMode
```
Comment (jeffswartz) [inline src/OpenTok/Broadcast.php:38]: I think we should create a StreamMode class with constants. (See the ArchiveMode class.)

**Review by jeffswartz — CHANGES_REQUESTED** [review]

**Comment by jeffswartz** [comment]
Note that I targeted the dev branch, instead of main.

File: src/OpenTok/Archive.php [hunk 2]
```diff
@@ -49,6 +49,10 @@
 * @property string $size
 * The size of the MP4 file. For archives that have not been generated, this value is set to 0.
 *
+* @property string $streamMode
+* Allows for an archive stream to be manually updated to add streams individually.
+* Default is set to <code>auto</code>.
```
Comment (jeffswartz) [inline src/OpenTok/Archive.php:54]: See the description for the `streamMode` option for the REST method for starting an archive: https://tokbox.com/developer/rest/#start_archive. Note that we need to add methods for adding and removing streams from archives (that use `streamMode` manual) to the PHP SDK (see this REST API method: https://tokbox.com/developer/rest/#selecting-archive-streams).

The same comment applies to the `streamMode` for broadcasts.

**Review by jeffswartz — CHANGES_REQUESTED** [review]

File: src/OpenTok/Archive.php (hunk 2 above)
Comment (SecondeJK) [reply src/OpenTok/Archive.php:54]: These methods have now been added

**Review by jeffswartz — APPROVED** [review]

**Pull request merged**